src/frequenz/client/dispatch/__main__.py

```diff
@@ -86,7 +86,8 @@ def print_dispatch(dispatch: Dispatch) -> None:
     # Format the target
     if dispatch.target:
         if len(dispatch.target) == 1:
-            target_str: str = str(dispatch.target[0])
+            (el,) = dispatch.target
+            target_str: str = str(el)
```

Comment:
This change is because `target` is a frozenset now

Reply:
You cannot index-access a set, it has no order :)

`el` stands for `element`.. sure I'll make it more clear :smile: 
I was surprised as well, but this seems to be the easiest way to access the only element of a set with size 1